spec/helpers/application_helper_spec.rb

```diff
@@ -225,6 +225,11 @@
       expect(subject).to eq(helper.url_for_db(helper.controller_for_vm(helper.model_for_vm(@record)), @action))
     end
 
+    it "when record is ManageIQ::Providers::AutomationManager" do
+      @record = ManageIQ::Providers::AutomationManager.new
```

Comment:
Should this be testing against base classes or the leaf classes that will actually be created?